### DISCUSSION TITLE
Boards with two Voltronic inverters And 4 batteries in parallel

### DISCUSSION THREAD

**Discussion by saviothecnic** [slipx06/sunsynk-power-flow-card#298] [General]
I discovered this amazing visualization
I note that I don't see any Voltronic Power brand inverters in the list
which is very famous and widespread
Are you thinking of adding it? maybe the KING series
It would also be nice to see the inverter icon> graphically
The operating logic of the Inverter
It is the source priority

Question I have two single-phase inverters in parallel
and 4 batteries always in parallel you have an option to graphically see 2 inverters in parallel and 4 batteries in parallel
and the display of a 5th battery which is the mathematical average of the 4 batteries displayed individually.

Thank you

**Comment by Pho3niX90** [comment]
To answer the first question. 
Technically all imverters are supported out of the box, but to have the imagr of the inverter displayed, it needs to be added. Do you have a image? 

For the last one, there isn't space to fit so many objects into the card currently. Will maybe see how lit reacts to dynamic scaling, however, the animations might break.

**Reply by saviothecnic** [reply]
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/6697807/8e1c8b1f-6f40-4532-8856-542642d0b6a0)


Sure place image taken from the manufacturer's website
but if you need higher resolution I can post a real photo of mine

Thanks for the quick response
I will be happy to try if you can implement this thing
